fontir/src/variations.rs

```diff
@@ -39,6 +44,17 @@ pub struct VariationModel {
     delta_weights: Vec<Vec<(usize, OrderedFloat<f32>)>>,
 }
 
+fn only_known_axes(
```

Comment:
this looks like it wants to a method called `retain` on `NormalizedLocation`, which would be similar to existing `std` apis for filtering items from a collection in place.

Reply:
I agree! Updating so.